src/toast/examples.js

```diff
@@ -124,11 +103,67 @@ class NotificationExample extends React.Component<{}, {toasts: []}> {
         >
           Negative toast
         </Button>
-        <Centered>
-          {this.state.toasts.map((toastOptions, index) => {
-            return <Toast key={index} {...toastOptions} />;
-          })}
-        </Centered>
+      </div>
+    );
+  }
+}
+
+class ToasterAdvancedExample extends React.Component<{}, {cleared: boolean}> {
+  keyToUpdate: React.Key;
+  state = {cleared: false};
+
+  componentDidMount() {
+    this.addNotifications();
+  }
+
+  addNotifications() {
+    toaster.info('Info notification', {closeable: false});
+    toaster.positive(<span>Positive notification</span>);
+    // $FlowFixMe
```

Comment:
Are you planning to fix the flow stuff in this diff or differing to a subsequent diff?